Compile error

**Please state your issue using the following template and, most importantly, in English.**

**Describe the bug**
```
[2020-10-10T01:36:01.091Z] ptxas warning : Value of threads per SM for entry _ZN5faiss3gpu15pqCodeDistancesIfLi2ELb0EEEvNS0_6TensorIfLi2ELb1EiNS0_6traits16DefaultPtrTraitsEEEiS5_NS2_IfLi3ELb1EiS4_EENS2_IiLi2ELb1EiS4_EENS2_IT_Li4ELb1EiS4_EE is out of range. .minnctapersm will be ignored

[2020-10-10T01:36:01.091Z] ptxas warning : Value of threads per SM for entry _ZN5faiss3gpu15pqCodeDistancesIfLi2ELb1EEEvNS0_6TensorIfLi2ELb1EiNS0_6traits16DefaultPtrTraitsEEEiS5_NS2_IfLi3ELb1EiS4_EENS2_IiLi2ELb1EiS4_EENS2_IT_Li4ELb1EiS4_EE is out of range. .minnctapersm will be ignored

[2020-10-10T01:36:01.091Z] ptxas warning : Value of threads per SM for entry _ZN5faiss3gpu15pqCodeDistancesIfLi1ELb0EEEvNS0_6TensorIfLi2ELb1EiNS0_6traits16DefaultPtrTraitsEEEiS5_NS2_IfLi3ELb1EiS4_EENS2_IiLi2ELb1EiS4_EENS2_IT_Li4ELb1EiS4_EE is out of range. .minnctapersm will be ignored

[2020-10-10T01:36:01.091Z] ptxas warning : Value of threads per SM for entry _ZN5faiss3gpu15pqCodeDistancesIfLi1ELb1EEEvNS0_6TensorIfLi2ELb1EiNS0_6traits16DefaultPtrTraitsEEEiS5_NS2_IfLi3ELb1EiS4_EENS2_IiLi2ELb1EiS4_EENS2_IT_Li4ELb1EiS4_EE is out of range. .minnctapersm will be ignored

[2020-10-10T01:36:01.091Z] ./faiss/gpu/impl/PQCodeDistances-inl.cuh(57): error: identifier "loadingThreads" is undefined

[2020-10-10T01:36:01.091Z] 

[2020-10-10T01:36:01.092Z] 1 error detected in the compilation of "/tmp/tmpxft_00003ab2_00000000-4_PQCodeDistances.cpp4.ii".

[2020-10-10T01:36:01.092Z] make[3]: *** [gpu/impl/PQCodeDistances.o] Error 2

[2020-10-10T01:36:01.092Z] make[2]: *** [src/index/faiss_ep-prefix/src/faiss_ep-stamp/faiss_ep-build] Error 2

[2020-10-10T01:36:01.092Z] make[1]: *** [src/index/CMakeFiles/faiss_ep.dir/all] Error 2

[2020-10-10T01:36:01.092Z] make: *** [all] Error 2
```
**Steps/Code to reproduce behavior**
```
http://internal.zilliz.com:18080/jenkins/blue/organizations/jenkins/milvus-ci/detail/PR-3725/19/pipeline

```
**Expected behavior**
A clear and concise description of what you expected to happen.

**Method of installation**
- [ ] Docker/cpu
- [ ] Docker/gpu
- [ ] Build from source

**Environment details**
- Hardware/Softwars conditions (OS, CPU, GPU, Memory)
on GPU 

- Milvus version (master or released version)
0.11.0

**Configuration file**
Settings you made in `server_config.yaml` or `milvus.yaml`

```yaml
paste-file-content-here
```

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Additional context**
Add any other context about the problem here.

--- a/core/src/index/thirdparty/faiss/gpu/impl/PQCodeDistances-inl.cuh
+++ b/core/src/index/thirdparty/faiss/gpu/impl/PQCodeDistances-inl.cuh
@@ -52,6 +52,15 @@ pqCodeDistances(Tensor<float, 2, true> queries,
   auto subQuantizer = blockIdx.y;
 
 
+  // Each thread will load the pq centroid data for the code that it
+  // is processing
+  if(!loadingThreads) {
+#pragma unroll
+      for (int i = 0; i < DimsPerSubQuantizer; ++i) {
+        subQuantizerData[i] = pqCentroids[subQuantizer][i][code].ldg();
+      }
+  }
+
   // Where we store our query vector
   float* smemQuery = smem;
 
@@ -147,12 +156,6 @@ pqCodeDistances(Tensor<float, 2, true> queries,
         }
       } else {
 
-        // Each thread will load the pq centroid data for the code that it
-        // is processing
-        #pragma unroll
-        for (int i = 0; i < DimsPerSubQuantizer; ++i) {
-        subQuantizerData[i] = pqCentroids[subQuantizer][i][code].ldg();
-        }
 
         // These are the processing threads
         float dist = 0.0f;

--- a/core/src/index/thirdparty/faiss/gpu/impl/PQCodeDistances-inl.cuh
+++ b/core/src/index/thirdparty/faiss/gpu/impl/PQCodeDistances-inl.cuh
@@ -54,7 +54,7 @@ pqCodeDistances(Tensor<float, 2, true> queries,
 
   // Each thread will load the pq centroid data for the code that it
   // is processing
-  if(!loadingThreads) {
+  if(!isLoadingThread) {
 #pragma unroll
       for (int i = 0; i < DimsPerSubQuantizer; ++i) {
         subQuantizerData[i] = pqCentroids[subQuantizer][i][code].ldg();

